docs/src/pages/guide/how-to/01-creating-agents.mdx

```diff
@@ -123,46 +147,56 @@ main();
 
 Or, you can [stream structured response](../reference/agents/streamObject.mdx):
 
-```ts showLineNumbers copy
-const query4 =
-  "I want to make lasagna, can you generate a lasagna recipe for me?";
-console.log(`Query: ${query4}`);
-
-const lasagnaStreamedResponse = await chefAgent.streamObject({
-  messages: [query4],
-  structuredOutput: {
-    ingredients: {
-      type: "array",
-      items: {
-        type: "object",
+```ts showLineNumbers filename="src/mastra/index.ts" lines={8-29} /streamObject/ copy
+async function main() {
+  const query4 =
+    "I want to make lasagna, can you generate a lasagna recipe for me?";
+  console.log(`Query: ${query4}`);
+
+  const lasagnaStreamedResponse = await chefAgent.streamObject({
+    messages: [query4],
+    structuredOutput: {
+      ingredients: {
+        type: "array",
         items: {
-          name: {
-            type: "string",
-          },
-          amount: {
-            type: "number",
+          type: "object",
+          items: {
+            name: {
+              type: "string",
+            },
+            amount: {
+              type: "number",
+            },
           },
         },
       },
-    },
-    steps: {
-      type: "array",
-      items: {
-        type: "string",
+      steps: {
+        type: "array",
+        items: {
+          type: "string",
+        },
       },
     },
-  },
-});
+  });
 
-console.log("\n👨‍🍳 Chef Michel: ");
+  console.log("\n👨‍🍳 Chef Michel: ");
 
-// Handle the stream
-for await (const chunk of lasagnaStreamedResponse.textStream) {
-  // Write each chunk without a newline to create a continuous stream
-  process.stdout.write(chunk);
+  // Handle the stream
+  for await (const chunk of lasagnaStreamedResponse.textStream) {
+    // Write each chunk without a newline to create a continuous stream
+    process.stdout.write(chunk);
+  }
+
+  console.log("\n\n✅ Recipe complete!");
 }
 
-console.log("\n\n✅ Recipe complete!");
+main();
+```
+
+Run the code with:
+
+```bash copy
+pnpx bun src/mastra/index.ts
```

Comment:
should be `npx tsx`

Reply:
`tsx` compiles to JS, and requires some configuration to get up and working.
`bun` comes shipped with a ts runner.

Reply:
okay great. let's standardize on `npx bun` then across the docs

Reply:
Got it.